l10n.js

```diff
@@ -219,40 +223,70 @@ document.webL10n = (function(window, document, undefined) {
               parsedRawLinesCallback();
               return;
             }
-            var line = entries.shift();
```

Comment:
I suggested to change your PR and use the original code with the four lines that I had mentioned, because that is less complex (and easier to review) than your current alternative. There is already at least one bug in your PR: It does not work when `@import` is used and asynchronous loading is enabled (which is the default setting). To avoid any other regressions, please revert back to the original code and add the lines that I suggested.

Since you don't want to support escapes in comments (which I agree with), put those four lines below, i.e. let the code be something like this:

```js
// comment or blank line?
if (reComment.test(line))
  continue;

if (entries.length && line.charAt(-1) === '\\') {
  entries[0] = line.slice(0, -1) + entries[0].replace(/^\s+/, '');
  continue;
}
```

Note that I also added left-trimming of whitespace.
And the code doesn't account for the fact that lines could end with multiple backslashes, and how to handle them.